.eslintrc.json

```diff
@@ -3,6 +3,7 @@
         "es6": true,
         "node": true
     },
+    "parser": "babel-eslint",
```

Comment:
eslint didn't like async functions without this. But it looks like a better way of configuring that is the following, which I'll change to.

```
    "parserOptions": {
        "ecmaVersion": 2017
    },
```